octoprint_mrbeam/static/js/mrbeam.js

```diff
@@ -499,9 +519,14 @@ $(function () {
                             // give it some time to settle in an endless loop
                             setTimeout(function () {
                                 OctoPrint.socket.reconnect();
+                                // Add to analytics to check how often passive logins are used after 401 errors
+                                self.analytics.send_fontend_event(
+                                    "passive_login",
```

Comment:
Since you add new events, please increase the version number of analytics in analytics handler and also add these to this documentation: https://docs.google.com/document/d/1G2UETo65dsFy_X6OhoSsyo0XIFSgjfoFy-n-5RSYJPg/edit